docs/components/page-higher-order/withDocumentationElements.js

```diff
@@ -1,28 +1,29 @@
-import * as React from 'react';
+import { MDXProvider } from '@mdx-js/tag';
```

Comment:
As of MDX 1, this should be `@mdx-js/react` 😄 

You should be able to change this without breaking changes!

Reply:
Yeah, I know, I need to sync it with Your PR some way 😁